src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/OperatorsShouldHaveSymmetricalOverloadsTests.cs

```diff
@@ -145,11 +142,11 @@ public async Task CSharpTestOperatorType()
             await VerifyCS.VerifyAnalyzerAsync(@"
```

Comment:
💡 This one should have a test name or comment indicating that the test intentionally uses invalid code